docs/howto/acuity_ramp.md

```diff
@@ -0,0 +1,122 @@
+[_Value Density_](../reference/decision_points/value_density.md) might be more readily discerned than
+[_Mission Impact_](../reference/decision_points/mission_impact.md) for some stakeholders because it's easier to 
+count how many of something there are than to estimate the impact of a loss of specific instances of the thing.
+An organization might start with _Value Density_ as a proxy for _Mission Impact_ and then, as they develop a better
+understanding of their environment, they might replace _Value Density_ with _Mission Impact_ in their decision model.
+
+
+## An Acuity Ramp in Action
+
+The _acuity ramp_ idea is a way to show how a stakeholder could "grow into" their desired decision function as their
+data collection and analysis capabilities increase. We demonstrate this with the following example.
+
+!!! example "An Acuity Ramp for a Growing System Deployer Organization"
+
+    A system deployer organization with few initial resources might start with a simple decision model that only includes a custom
+    `IN_KEV` decision point. The `IN_KEV` decision point would be a simple binary indicator of whether a vulnerability
+    has been added to CISA's [Known Exploited Vulnerabilities](https://www.cisa.gov/known-exploited-vulnerabilities-catalog)
+    ([KEV](https://www.cisa.gov/known-exploited-vulnerabilities-catalog)) catalog. Because this information is free,
+    publicly available, and because it is a simple binary indicator, it is easy to gather and analyze even for a very
+    small organization.
+
+    The following table shows how the organization might expand their decision model as they grow in capability.
+    
+    | Acutiy Level | Tree |
+    | --- | --- |
+    | 1 | ```[IN_KEV]``` |
+    | 2 | ```[EXPLOITATION_1]``` |
+    | 3 | ```[EXPLOITATION_1, SYSTEM_EXPOSURE_1_0_1]``` |
+    | 4 | ```[EXPLOITATION_1, SYSTEM_EXPOSURE_1_0_1, AUTOMATABLE_2]``` |
+    | 5 | ```[EXPLOITATION_1, SYSTEM_EXPOSURE_1_0_1, AUTOMATABLE_2, MISSION_IMPACT_2, SAFETY_IMPACT_1]```
+
+    !!! tip "Acuity Levels are Stakeholder-Specific" 
+
+        This example is demonstrating the _concept_ of _acuity levels_ in SSVC adoption. We are specifically
+        _not_ saying that there are 5 levels of acuity in SSVC; in principle this concept could be applied to any number
+        of levels (including just one). In practice, the number of levels necessary would be a stakeholder-specific implementation choice, so 
+        there is no inherent meaning to the "Acuity Levels" shown here outside the context of this example.
+    
+    The remainder of this example shows one path the organization might take to grow their decision model
+    according to the table above.
+
+    ### Improved Exploit Awareness (Acuity Level 2)
+
+    As the organization becomes more capable of gathering and analyzing data, they might start collecting their own
+    data on exploitation, allowing them to move to a more complex decision tree that replaces the binary `IN_KEV` 
```

Comment:
How do you feel about referencing other public information sources about exploitation information or predictions here?